packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -1856,4 +1857,20 @@ describe('RpcContractProvider test', () => {
       done();
     });
   });
+
+  describe('Testing Bug #1762 mapTypecheckError', () => {
```

Comment:
Try and use the name of the member method/variable as the describe block name. In this case, it should be something like `describe('storage', ...`

describe blocks should be quite high level and general sounding